setuptools/dist.py

```diff
@@ -603,7 +603,7 @@ def _expand_patterns(patterns):
         return (
```

Comment:
If you want reproducibility, why not sort the filtered result? Of course, it'll turn a generator expression into a list or something...

Reply:
I prefer this minimal approach that retains the order of each pattern. If you were to sort the entire result, you would lose fidelity about the order in which the patterns were supplied. Thus if the user specifies LICENSE_B, LICENSE_A as separate globs, they'll get them in that order.